servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,40 @@
\ No newline at end of file
+NOTE: There are some factors that influence ServiceControl resource usage and performance that are less likely to change between environments but are specific to the system being monitored such as choice of transport and average message size. 
+
+
+## Specific Considerations
+
+- Each environment should have a dedicated ServiceControl instance.
+- In production, ServiceControl should run on a **dedicated machine**.
+- Turn off [Message Auditing](/nservicebus/operations/auditing.md) if it is not needed. The audit ingestion capability of ServiceControl is primarily to support system analysis with ServiceInsight. If ServiceInsight is not deployed in an environment then configure ServiceControl to read audit messages from an empty queue or turn off messaging auditing for each endpoint. Message auditing may be important for some endpoints but not other others. 
+- Turn off [Audit Forwarding](/servicecontrol/errorlog-auditlog-behavior.md) if it is not needed. ServiceControl sends a copy of each audited message to configured Audit Forwarding queue. If these messages are not being used, turn this feature off.
+- Plugins communicate between the Endpoint they installed in to ServiceControl using messaging and the configured transport of your system. Each instance of a plugin adds more messages to the ServiceControl queue which can delay the processing of each message and make ServiceControl less responsive.
+  - [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) - Not all endpoints are mission critical and need to be monitored with heartbeats. For Endpoints that do need to be monitored, adjust the Heartbeat Interval. Increasing the interval ensures that ServiceControl is able to process heartbeats in a timely manner but also increases the window that an Endpoint can be unavailable for before being detected by ServiceControl. Heartbeat messages tend to be frequent and a large backlog can occur if ServiceControl is offline for an extended period. When this happens, it can take ServiceControl some time to process old heartbeats when it restarts. 
+  - [Saga Audit](/servicecontrol/plugins/saga-audit.md) - the saga audit plugin produces a lot of data. It's use outside of a development environment is not recommended.
+- Exclude the ServiceControl [database directory](/servicecontrol/configure-ravendb-location.md) from anti-virus checks.
+- Do not downgrade releases of ServiceControl. ServiceControl uses an embedded database and changes to the internal data structures can occur between releases. Rolling back may cause index corruption or data loss. Perform testing in a lower environment before upgrading in production environments.
```

Comment:
I don't know. It was a part of @petersgiles content and I didn't want to lose it.